kubernetes/loculus/values.yaml

```diff
@@ -593,17 +593,29 @@ defaultOrganismConfig: &defaultOrganismConfig
       - name: insdcAccessionFull
         displayName: INSDC accession
         customDisplay:
-          type: link
-          url: "https://www.ncbi.nlm.nih.gov/nuccore/__value__"
+          type: linkWithMenu
+          linkMenuItems:
+            - name: View in NCBI
+              url: "https://www.ncbi.nlm.nih.gov/nuccore/__value__"
+            - name: View in ENA
+              url: "https://www.ebi.ac.uk/ena/browser/view/__value__"
+            - name: View in DDBJ
+              url: "https://getentry.ddbj.nig.ac.jp/getentry/na/__value__"
         header: "INSDC"
         ingest: genbankAccession
         noInput: true
         perSegment: true
         oneHeader: true
       - name: bioprojectAccession
         customDisplay:
-          type: link
-          url: "https://www.ncbi.nlm.nih.gov/bioproject/__value__"
+          type: linkWithMenu
+          linkMenuItems:
+            - name: View in NCBI
+              url: "https://www.ncbi.nlm.nih.gov/bioproject/__value__"
+            - name: View in ENA
+              url: "https://www.ebi.ac.uk/ena/browser/view/__value__"
+            - name: View in DDBJ
```

Comment:
I am not sure this link works I see: 
<img width="1426" height="256" alt="image" src="https://github.com/user-attachments/assets/c109f174-a6f9-4978-b57a-e3b258a95d78" />

Reply:
I also see this error for biosamples in DDBJ

Reply:
Thanks for catching these - hopefully fixed let's see